_posts/2022-06-28-2022-survey-results.md

```diff
@@ -0,0 +1,146 @@
+
+## How do you perceive OpenRefine?
+
+Which features make you choose OpenRefine over other tools? Many people mention that they appreciate OpenRefine's GUI, price (free!), flexibility, power, reconciliation features, and relative ease of use.
+
+<a href="https://raw.github.com/OpenRefine/openrefine.github.com/master/images/2022survey/2022-survey-wordcloud-why.png"><img src="https://raw.githubusercontent.com/OpenRefine/openrefine.github.com/master/images/2022survey/2022-survey-wordcloud-why.png" class="inset" /></a>
+
+Which tool(s) would you use if OpenRefine would not be available to you? Excel is a winner here, and quite a few people also mention Python and R. QuickStatements would be an important alternative for Wikimedia and Wikibase users. One user mentions they would "sob the entire time", which we of course want to prevent.
+
+<a href="https://raw.github.com/OpenRefine/openrefine.github.com/master/images/2022survey/2022-survey-wordcloud-alternatives.png"><img src="https://raw.githubusercontent.com/OpenRefine/openrefine.github.com/master/images/2022survey/2022-survey-wordcloud-alternatives.png" class="inset" /></a>
+
+How do you describe OpenRefine to someone else? Many of the descriptions involve the words "data", "powerful" and "cleaning", and we very much appreciate phrases like "spreadsheets on steroids", "the Ferrari of spreadsheets", "a librarian's dream", "swiss army knife", or simply "magic".
+
+<a href="https://raw.github.com/OpenRefine/openrefine.github.com/master/images/2022survey/2022-survey-wordcloud-describe.png"><img src="https://raw.githubusercontent.com/OpenRefine/openrefine.github.com/master/images/2022survey/2022-survey-wordcloud-describe.png" class="inset" /></a>
+
+We received many feature requests as answers to the prompt "It would be awesome if OpenRefine...". Quite a few of these major requests are very familiar to us, and also mentioned on our [roadmap](https://github.com/OpenRefine/OpenRefine/wiki/Roadmap).
+
+* Support for large datasets with many columns and/or rows (3 requests). Good news: we are working towards this goal for OpenRefine's major new 4.x release.
+* A better UX (3 requests), which makes it easier for newcomers to use OpenRefine (3 requests).
+* A free online instance of OpenRefine / hosted OpenRefine (3 requests).
+* Multi-user support in OpenRefine (2 requests).
+* Better Python support (3).
+* Allow working with R for syntax based work (2).
+* More 'point and click' functions to replace GREL (2).
+* Some simple data visualizations (2 requests), including the possibility to plot georeferenced data on a map.
+* Easier import from (2 requests) and reconciliation with external datasets via APIs
+* A feature to add new rows (2)
+* Better developed, more explicit and more detailed notifications and warnings (2)
+
+Some feature requests are specific to Wikimedia, Wikibase and Wikidata support:
+    * Faster Wikidata upload
```

Comment:
The indent here, and the two rows below, breaks the bullet point list when I look at the rendered markdown.

Reply:
I think I did a new commit which fixes this? LMK if I should do something differently.